### PR TITLE
Cherry-pick #3924 to 5.x: heartbeat: setup default ports in http monitors

### PR DESCRIPTION
**Pull request by 7AC**
Cherry-pick of PR #3924 to 5.x branch. Original message: 

Default http to 80 and https to 443

Resolves https://github.com/elastic/beats/issues/3915

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -123,6 +123,8 @@ https://github.com/elastic/beats/compare/v5.2.2...v5.3.0[View commits]
 *Filebeat*
 
 - Fix empty registry file on machine crash. {issue}3537[3537]
+*Heartbeat*
```
Comment (ruflin) [inline CHANGELOG.asciidoc]: a newline is needed on top

**Comment by ruflin** [comment]
@7AC I recommend you to cherry pick the squashed commit (https://github.com/elastic/beats/commit/77c5b7415b45bdfb416922ea793cd4973f79bab4) instead of getting 3 commits.

I think the travis failure is not related to this PR.

File: CHANGELOG.asciidoc (hunk 1 above)
Comment (7AC) [inline CHANGELOG.asciidoc]: Ah interesting, no idea what happened

**Comment by 7AC** [comment]
@ruflin I tried with the squashed commit first but the script didn't handle it too well (lots of unrelated commits ended up in the PR)

**Pull request merged**